src/services/authentication/strategies/system/verifier.js

```diff
@@ -11,50 +11,69 @@ class SystemVerifier {
 		this.verify = this.verify.bind(this);
 	}
 
+	//Create LDAP username
+	_getUsername({ username, password, systemId, strategy, schoolId }) {
+		return this.app.service('schools').get(schoolId)
+			.then(school => {
+				if (strategy == 'ldap') {
+					return school.ldapSchoolIdentifier + '/' + username;
```

Comment:
@Metauriel 	@malcm 	revisiting this, can we come up with something more transparent, so we don't need to hardcode the ldap strategy here?